maintainers/maintainer-list.nix

```diff
@@ -13195,7 +13195,6 @@
     github = "matthewpi";
     githubId = 26559841;
     name = "Matthew Penner";
-    keys = [ { fingerprint = "5118 F1CC B7B0 6C17 4DD1  5267 3131 1906 AD4C F6D6"; } ];
```

Comment:
why?

Reply:
Read the description of the commit. My commits may be signed, but it's not with the key listed there, no reason to leave that around.